### PR TITLE
Import utils from fink-broker

### PR DESCRIPTION
**Pull request by FusRoman**
This PR brings all utilities that were stored in fink-broker. The idea is to use them easily outside fink-broker.

### CODE REVIEW

**Comment by FusRoman** [comment]
Origin of the new utility functions :
* All the files in the fink-utils/broker folder come from fink-broker/fink_broker
* The file fink-utils/science/utils.py come from the file fink-broker/fink_broker/science (I took only the functions related to healpix, I left the functions apply_science_module into fink-broker)
* The file fink-utils/test/tester.py contains the test functions from fink-broker and one function from fink-science. I rename these functions regarding the origin.

File: .github/workflows/linter.yml [hunk 1]
```diff
@@ -26,4 +26,4 @@ jobs:
         pip install flake8
     - name: Lint utilities
       run: |
-        flake8 fink_utils/*/*.py --count --show-source --statistics --ignore=E302,E501
+        flake8 ../fink-utils
```
Comment (JulienPeloton) [inline .github/workflows/linter.yml:29]: Why do you need to change the path?

**Review by JulienPeloton — APPROVED** [review]
Thanks! I only left one comment, and then we can merge.

File: .github/workflows/linter.yml (hunk 1 above)
Comment (FusRoman) [reply .github/workflows/linter.yml:29]: I added a .flake8 file that specifies the flake8 options without putting each option in the command line, but you need to run flake8 from the project root folder. flake8 will discover the .flake8 file and parse it.

Comment (FusRoman) [reply .github/workflows/linter.yml:29]: flake8 is also able to discover each .py files. So, we can remove the /*/*.py glob.

Comment (JulienPeloton) [reply .github/workflows/linter.yml:29]: OK!

**Pull request merged**